### PR TITLE
feat(gov-0.1): first version

### PR DESCRIPTION
**Pull request by qbzzt**
As requested by Bobby (https://optimism-workspace.slack.com/archives/C0214EMUUJ3/p1652828078537819).

Closing: INT-599

### CODE REVIEW

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -141,6 +142,7 @@ module.exports = {
       ],
       '/docs/developers/': [
         '/docs/developers/releases.md',
+        '/docs/developers/bedrock.md',
```
Comment (Nicca42) [inline src/.vuepress/config.js:145]: This should not be in this PR. Please remove.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/governance/gov-01.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+| Governance Fund (Phase 1) | OP distributions to proactively incentivize future growth of projects and communities in the Optimism ecosystem. [Phase 1](gov-fund.md#phase-0) begins after Airdrop #1 |	Forum + Snapshot | 1 (~2 weeks)	| 10% | 51% |
+| Protocol Upgrade | Scheduled changes to the on-chain smart contracts comprising the mainnet Optimism protocol	| Snapshot | 2 (~4 weeks) | 10% | 51% |
+| Inflation Adjustment | Changes to the inflation rate of newly minted OP (currently capped at 2% annually) | Snapshot | 2 (~4 weeks) | 10% | 76% |
+| Director Removal | Removal of a director of the Optimism Foundation | Snapshot | 2 (~4 weeks) | 10% | 76% |
+| Treasury Appropriations | The amount of OP the Optimism Foundation may spend or distribute annually, beginning in Year 2 of its existence (the Year 1 budget is 30% of the initial total OP supply) | Proposals to be initiated by the Foundation | 2 (~4 weeks) | 10% | 51%
+| Rights Protections | OP holders must consent to any changes to the founding documents of the Optimism Foundation, if those changes would materially reduce their rights | Proposals to be initiated by the Foundation	| 2 (~4 weeks) | 10% | 51%
+
+Notes:
```
Comment (Nicca42) [inline src/docs/governance/gov-01.md:102]: Can we move this to above the table? Happy to merge after that.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/governance/gov-01.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/governance/gov-01.md:102]: [It's standard to put notes below the table](https://apastyle.apa.org/style-grammar-guidelines/tables-figures/tables). That's where I'd expect them and that's where I think most readers will expect them. Is there a reason not to follow that standard here?

Comment (Nicca42) [reply src/docs/governance/gov-01.md:102]: If that is convention then it's all good, just seems weird to me, but I am no doc expert.

**Review by Nicca42 — APPROVED** [review]
LGTM

**Pull request merged**